tests/integration/test_metadata_uri_templates.py

```diff
@@ -95,6 +101,7 @@ def test_prepare_configs_from_rule_binding(
             configs["configs"]["entity_configs"]["dataplex_location"] = "<your-gcp-dataplex-region-id>"
             configs["configs"]["entity_configs"]["dataplex_asset_id"] = "<your-gcp-dataplex-asset-id>"
             configs["configs"]["entity_configs"]["dataplex_createTime"] = "<dataplex_entity_createTime>"
+            configs["configs"]["entity_configs"]["dataplex_updateTime"] = "<dataplex_entity_updateTime>"
```

Comment:
Can we remove update time? I've decided to exclude this because it changes the config hashsum in unpredictable ways